lua/tests/docgen/docgen_spec.lua

```diff
@@ -0,0 +1,238 @@
+local docgen = require('docgen')
```

Comment:
This file does no longer exist it got split up in the `function_spec`, `brief_spec`, etc